docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -346,6 +346,7 @@ The `--set` CLI method is more appropriate for quick test deployments.
     auditLogMirrorOnStdout: false                   # Whether to mirror audit log entries to stdout in JSON format (useful for external log collectors)
     sessionRecordingBucket: teleport-helm-sessions  # S3 bucket to use for Teleport session recordings
     backups: true                                   # Whether or not to turn on DynamoDB backups
+    autoScaling: false                              # Whether Teleport should configure DynamoDB's autoscaling.
```

Comment:
```suggestion
    dynamoAutoScaling: false                        # Whether Teleport should configure DynamoDB's autoscaling.
```

I think I would change the name of this value to clarify that it's only DynamoDB autoscaling rather than any other form, just in case people see the value and assume it's a magic panacea.